Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM signiant/docker-jenkins-centos-base:centos7-java8
 MAINTAINER devops@signiant.com
 
+EXPOSE 8000
```

Comment:
Why do these ports have to be exposed?

Reply:
I think Dave made the change in 2018-10-15: Only expose the actual needed ports of it breaks the docker jenkins plugin with an object too big exception. 
But I think I duplicated the ports too. Will remove it